### ISSUE TITLE
CDN url

### ISSUE THREAD

**Issue by ismaelaarab** [NickNaso/hertzy#19]
Hi

Is it possible to use this library in the browser?

**Comment by NickNaso** [comment]
Hi @ismaelaarab,
at moment it's not possible use this module on the browser. I'm interested in your request. Could you explain how dow you want use it? From the title of the issue you want import it from a CDN like reported below:
`<script src="https://CDN-HOST/hertzy.js"></script>`
Is it right?

**Comment by ismaelaarab** [comment]
I want to use it in google appmaker. 

`<script src="https://CDN-HOST/hertzy.js"></script>` is exactly how i want to use it

**Comment by ismaelaarab** [comment]
Hey Nicola

Any updates on this?

**Comment by NickNaso** [comment]
Hi @ismaelaarab,
I started working on this issue and I found that I need to address the following task to end the work.

- [x] ~~Implementation of **EventEmitter** for the browser~~
- [x] ~~Loader that allow to choose the right implementation of the **EventEmitter** based on runtime (Browser, Node.js) the module is used.~~
- [x] ~~Create a build process using webpack and babel to produce the distribution of the module that is usable on Browser and Node.js at same time.~~

I will update the issue as I go forward with the work. Today I'm working on the EventEmitter implementation.

**Comment by NickNaso** [comment]
Hi @ismaelaarab, 
I worked on the possibility to produce a library that runs on browsers. My initial effort is done. For now you can import **hertzy** like reported below:

```html
<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Hertzy</title>
        <script src=" https://nicknaso.github.io/hertzy/hertzy.js"></script>
    </head>
    <body>
        <h1>Hertzy</h1>
        <p>Test the usage of Hertzy in the Browser</p>
        <script>
            // Get hertzy version and print it
            console.log(Hertzy.VERSION)
            // Get frequency to use
            const frequency = Hertzy.tune('user')
            // Listen 'user:add' event on 'user' frequency
            frequency.on('user:add', function (data) {
                // DO SOMETHING ...
                console.log('User added ...')
            })
             // Emit 'user:add' event on 'user' frequency
             frequency.emit('user:add', {
                username: 'NickNaso',
                password: '********',
                email: 'nicoladelgobbo@gmail.com'
            })
        </script>
    </body>
</html>
```
Now I have the first implementation, but I need to work on other things before to deliver new version of the module that will be available on a CDN.
In the specific I have to address at the following tasks:

- [ ] Check browser compatibility (IE is not supported for now)
- [ ] Improve the building process create two versions (development and minified)

Hope to update you soon. In the mean time could you try to use it and report some feedback? Thanks.

**Comment by mauro-d** [comment]
Hi @ismaelaarab, we have just released a version to use in the browsers.
If you are still interested, you can refer to this [guide](https://github.com/NickNaso/hertzy/tree/browser).

**Comment by NickNaso** [comment]
I'm closing the issue because it's solved when the PR **[#21](https://github.com/NickNaso/hertzy/pull/21)** has been landed.
